### ISSUE TITLE
Monorepo integratioon?

### ISSUE THREAD

**Issue by holgerd77** [ethereumjs/ethashjs#25]
After having slept a night on this I think it would actually a lot of sense to add this to the monorepo.

@evertonfraga What do you think? Would this be a substantial additional integration effort or is this relatively easily doable now?

Follow-up on #20

**Comment by evertonfraga** [comment]
That can be done rather easily now. Will work on it :)

**Comment by holgerd77** [comment]
Thanks, currently trying to ping @wanderer for publishing rights.

**Comment by holgerd77** [comment]
(still one of the old libraries where these never have been transferred)

**Comment by evertonfraga** [comment]
if we're planning to migrate it to the monorepo, it can be renamed to the scoped way: `ethashjs => @ethereumjs/ethash`.

**Comment by holgerd77** [comment]
@evertonfraga yeah, sure, you're right, that will also be a way, forgot about that. Would nevertheless be nice to at least deprecate on the old npm publishing site.
